src/appleseed/renderer/modeling/bsdf/lambertianbrdf.cpp

```diff
@@ -236,4 +239,28 @@ auto_release_ptr<BSDF> LambertianBRDFFactory::create(
     return auto_release_ptr<BSDF>(new LambertianBRDF(name, params));
 }
 
+//
+// MicrofacetLambertianBRDFFactory class implementation.
+//
+
+const char* MicrofacetLambertianBRDFFactory::get_model() const
+{
+    return MicrofacetModel;
+}
+
+Dictionary MicrofacetLambertianBRDFFactory::get_model_metadata() const
+{
+    return
+        Dictionary()
+            .insert("name", MicrofacetModel)
+            .insert("label", "Microfacet Glossy BRDF");
```

Comment:
The lambertian BRDF and Glossy BRDF got the same Label.

Reply:
Fixed.